Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -33119,4 +33119,20 @@
             <Volume value="0.000000,0.000000"/>
         </Layers>
     </CActorSound>
+    <CActorDoodad id="AP_CharCrashingFactory">
+        <On Terms="ActorCreation" Send="$Death NonLooping"/>
+        <On Terms="AnimDone; AnimName Death" Send="Destroy"/>
+        <On Terms="ModelEvent; ModelEventName CharCrashingBuildingDrop" Send="Create AP_Doodad_CharCrashingBuildingDrop"/>
+        <On Terms="ModelEvent; ModelEventName CharCrashingBuildingDrop" Send="Create AP_Doodad_CharCrashingBuildingFlames"/>
+        <On Terms="ModelEvent; ModelEventName CharCrashingBuildingImpact" Target="AP_Doodad_CharCrashingBuildingDrop" Send="Destroy"/>
+        <On Terms="ModelEvent; ModelEventName CharCrashingBuildingImpact" Target="AP_Doodad_CharCrashingBuildingFlames" Send="Destroy"/>
+        <On Terms="ModelEvent; ModelEventName CharCrashingBuildingImpact" Send="Create AP_Doodad_CharCrashingBuildingImpact"/>
+        <On Terms="ModelEvent; ModelEventName CharCrashingBuildingImpact" Send="Create AP_Doodad_CharCrashingBuildingImpactDebris"/>
+        <RandomScaleRange value="1.000000,1.000000"/>
+        <DoodadFlags index="PauseAnimsWhileFogged" value="0"/>
+    </CActorDoodad>
+    <CActorSound id="AP_Doodad_CharCrashingBuildingDrop" parent="SoundOneShot"/>
+    <CActorSound id="AP_Doodad_CharCrashingBuildingFlames" parent="SoundOneShot"/>
+    <CActorSound id="AP_Doodad_CharCrashingBuildingImpact" parent="SoundOneShot"/>
+    <CActorSound id="AP_Doodad_CharCrashingBuildingImpactDebris" parent="SoundOneShot"/>

```

Comment:
What are these actors for?

Reply:
They were for a failed attempt to recreate the Gates of Hell building crash, so that when Swann tries to send you hellions + factory as non-Terran, you get something other than Hellions and the factory just crashes and explodes.

...But I gave up on the idea so there are no more trigger references to it. I just forgot to delete the actor doodad. Will remove that now.